### ISSUE TITLE
Skin updates are not visible to the target player

### ISSUE THREAD

**Issue by TheoRetisch1** [Zrips/CMI#2519]
The _/cmi skin_ command updates the skin only on the main world correctly. On all other worlds the command will be executed but the skin won't update, _/cmi skin update_ won't help either. The skin updates itself only after a world change or after a relog.

Reproduction steps:
Server with two worlds:
1) Go to the !main_world and type _/cmi skin Santa_ -> The command is executed but there is no skin update visible
2) Change to the main world or relog to see the new skin.

It's a weird behavior, but I'm sure there is a technical reason. I tried to rule out everything else.

config:
```
Skins:
  Enabled: true
  AutoApply: true
  SteveOnOff: false
  RequireSpecificPerm: false
  SkinUpdateTimer: 60
  SkinRequestFrequency: 10
```
CMI 8.6.15.8
Paper 1.13.2 - 652

**Comment by TheoRetisch1** [comment]
*edit*
The skin updates itself immediately for other players, but not for yourself. To update the skin for your own client you have to relog.

**Comment by TheoRetisch1** [comment]
The bug also occurs in a slightly modified form also in version **8.6.20.9**: Other players see the skin change of a player immediately. The player with the changed skin only sees the change after a server restart, a relog or client restart does not help.

Paper 1.13.2 - 655 
CMI 8.6.20.9

**Comment by TheoRetisch1** [comment]
Fixed in version 8.6.23.5!
